packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -322,6 +323,91 @@ describe('cluster', () => {
     });
   });
 
+  test('cluster with inline parameter group', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA,
```

Comment:
Doesn't this test contradict this statement from the docs?

>    * You need to use a versioned engine to auto-generate a DBParameterGroup

Do the docs need updating?

Reply:
Seems that Aurora has an internal default version other engines do not have.